openedx/core/djangoapps/coursetalk/tests/test_helpers.py

```diff
@@ -0,0 +1,18 @@
+""" CourseTalk widget helpers tests """
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from openedx.core.djangoapps.coursetalk.helpers import get_coursetalk_course_key
+
+
+class GetCoursetalkCourseKeyTests(TestCase):
```

Comment:
It would also be good to test `update_view_context` (or whatever you decide to rename it) here.